ecs-cli/modules/cli/compose/entity/service/service.go

```diff
@@ -132,8 +145,23 @@ func (s *Service) LoadContext() error {
 		if loadBalancerName != "" {
 			s.loadBalancer.LoadBalancerName = aws.String(loadBalancerName)
 		}
+		s.loadBalancers = []*ecs.LoadBalancer{s.loadBalancer}
+		// s.loadBalancers = append(s.loadBalancers, s.loadBalancer)
```

Comment:
Maybe get rid of this?

Reply:
When a user wants to add one load balancers, we will need this to put into an array because the createServiceInput currently accepts an array of load balancers.

Reply:
I mean the line that gets commented out